rmf_traffic_ros2/src/rmf_traffic_ros2/schedule/MonitorNode.cpp

```diff
@@ -168,6 +183,7 @@ std::shared_ptr<rclcpp::Node> MonitorNode::create_new_schedule_node()
 {
   auto database = std::make_shared<Database>(mirror.value().fork());
   auto node = std::make_shared<rmf_traffic_ros2::schedule::ScheduleNode>(
+    1, // Bump the node edition by one
```

Comment:
The monitor node, as currently designed and implemented, can only handle a single fail over. That's why I just left this at `1`.

Reply:
I don't see why that would be the case. If a fail over happens, we could just launch another monitor node that's set up to spawn a new schedule node with a node edition of 2. All we'd need to do is expose this parameter so that users can set it from the launch file or command line.

Reply:
OK, I can see that argument. I don't think it's particularly feasible in practice because the launch instance that started the system wouldn't know about the node, but it is doable. I'll add a parameter.

Reply:
I can see two feasible strategies:
1. Any `MonitorNode` that gets launched after the first fail over will need to be launched manually by a human operator. The human could either input the next node edition or the new `MonitorNode` could listen for the current node edition and just increment it by 1.
2. We could pre-assign consecutive node editions to a bunch of `MonitorNode`s and launch them all at the same time. We would redesign the fail over conditions so that the `MonitorNode` will only active its schedule node if it believes that all of its predecessors have failed / timed out. This would probably be very tricky to get right, and it introduces a risk that multiple schedule nodes could get activated at the same time, which would be a disaster. So strategy (1) would probably be good for now.

Reply:
Done in cdaa2fd.